extensions/common/auth/auth-basic/src/main/java/org/eclipse/edc/api/auth/basic/ConfigCredentials.java

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.api.auth.basic;
+
+public record ConfigCredentials(String username, String vaultKey) {
```

Comment:
I'd either make this package-private, or put bit of documentation on it

Reply:
Done, I had to make package-private also the `BasicAuthenticationProvider` to avoid a warning about visibility, anyway the module has been deprecated as a whole.